python/paddle/fluid/tests/unittests/test_query_op.py

```diff
@@ -0,0 +1,33 @@
+#   Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import print_function
+
+import unittest
+import paddle
+from paddle.fluid import core
+
+
+class TestCudnnVersion(unittest.TestCase):
+    def test_no_cudnn(self):
```

Comment:
this can be merged into the test below, since you can not test both case in one go.

Reply:
Done